files/acme-controller-template.yml

```
@@ -143,10 +147,10 @@ objects:
       spec:
         containers:
         - name: acme-controller
-          image: acme-controller:latest
+          image: ${DOCKER_IMAGE}:${DOCKER_IMAGE_TAG}
```

This needs to be the name/tag of the ImageStream which is `acme-controller:latest`.
I'd propose we change the tag name of the ImageStream (on line https://github.com/appuio/ansible-role-openshift-acme/pull/12/files#diff-20146561012af9a199100401e144b8e4R128) to `${DOCKER_IMAGE_TAG}` so this will become
```suggestion
          image: acme-controller:${DOCKER_IMAGE_TAG}
```

---

Right, forgot to change that back after re-adding the image stream